src/util/token.js

```diff
@@ -3,17 +3,9 @@ const crypto = require('crypto') // sodium doesn't have HMAC
 const base64url = require('base64url')
 
 const TOKEN_HMAC_INPUT = 'token'
+const AUTH_HMAC_INPUT = 'authorization'
 
-const token = (seed, publicKey) => {
-  // seed and public key should be stored as base64url strings
-  const seedBuffer = base64url.toBuffer(seed)
-  const publicKeyBuffer = base64url.toBuffer(publicKey)
-
-  const sharedSecretBuffer = tweetnacl.scalarMult(
-    crypto.createHash('sha256').update(seedBuffer).digest(),
-    publicKeyBuffer
-  )
-
+const tokenFromSharedSecret = (sharedSecretBuffer) => {
```

Comment:
I'm ok with doing it this way now because this change is an urgent improvement, but IMHO we should create a follow-up ticket to untangle ilp-plugin-virtual from ilp-kit.

(specifically, I think many connectors will want to run something like the "connector-rpc" server from the boogaloo demo, instead of ilp-kit. So either we make something like connector-rpc run directly behind nginx inside ilp-kit, or we create a stand-alone 'duplicated ledger' server as @emschwartz suggested.)